sktime/utils/datetime.py

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
+"""
+add comment to in align with the doc-quality
```

Comment:
This should say something descriptive - e.g., `pandas.datetime related utilities`